scss/_regions/_modal.scss

```diff
@@ -25,13 +25,15 @@
 // See http://www.github.com/DoSomething/modal
 [data-modal] {
   .heading.-banner {
-    // make space for "x" button and add proper left spacing
-    padding-left: $base-spacing;
-    padding-right: 72px;
+    // make space for "x" button on the right.
+    padding: $base-spacing 72px $base-spacing $base-spacing;
```

Comment:
Oh hah, didn't realize I included that commit in here. :blush: Yes, that's an adjustment based on the changes to the "banner" modifier in #536.